### ISSUE TITLE
Nuxt server startup issue - Cannot read property 'public' of undefined

### ISSUE THREAD

**Issue by kadlinobit** [Intevel/nuxt-directus#73]
### Version

nuxt-directus: 3.0.3
"@nuxt/bridge": "0.10.1",
"@nuxt/bridge-edge": "3.0.0-27296423.f3082ca",
"nuxt-edge": "2.16.0-27300977.ab1c6cb4",

### Reproduction Link
https://codesandbox.io/s/nuxt-bridge-directus-module-problem-tklbm2?file=/nuxt.config.js

### Steps to reproduce
1. Add directus as a dependency.
2. Set up directus in nuxt.config.js according to docs. 
```
  modules: ["nuxt-directus"],
  directus: {
    url: "http://localhost:8055",
    fetchUserParams: {
      fields: ["*", "djs.*"]
    },
    autoFetch: true
  }
```
3. Run dev server.

### What is Expected?
Dev server starts.

### What is actually happening?
There is an error:
```
Cannot read property 'public' of undefined 
  at setup (node_modules/nuxt-directus/dist/module.mjs:53:32)
  at ModuleContainer.normalizedModule (node_modules/nuxt-directus/node_modules/@nuxt/kit/dist/index.mjs:637:29)
  at async ModuleContainer.addModule (node_modules/@nuxt/core-edge/dist/core.js:174:20)
  at async ModuleContainer.ready (node_modules/@nuxt/core-edge/dist/core.js:47:5)
  at async Nuxt._init (node_modules/@nuxt/core-edge/dist/core.js:346:5)
```
### Additional info
It seems like nuxt-directus is expecting this to be in the `nuxt.config.js`:
```
runtimeConfig: {
   public: {}
}
```
When I add this to the `nuxt.config.js` (empty public object), server starts, but then there is a different error when app is loaded in the browser: 
```
[nuxt] [request error] Cannot read properties of undefined (reading 'autoFetch')
  at ./.nuxt/dist/server/server.js:13441:23  
  at callWithNuxt (./.nuxt/dist/server/server.js:8196:20)  
  at ./.nuxt/dist/server/server.js:8206:26  
  at createApp (./.nuxt/dist/server/server.js:1701:87)  
  at async module.exports../.nuxt/server.js.__webpack_exports__.default (./.nuxt/dist/server/server.js:4303:7)  
  at async Object.renderToString (./node_modules/vue-bundle-renderer/dist/runtime.mjs:172:19)  
  at async ./.nuxt/dev/index.mjs:533:21  
  at async ./.nuxt/dev/index.mjs:107:22  
  at async ./node_modules/h3/dist/index.mjs:592:19  
  at async Server.nodeHandler (./node_modules/h3/dist/index.mjs:538:7)
```

**Comment by Intevel** [comment]
@kadlinobit Do you know if this bug is only present on Nuxt Bridge?

**Comment by Intevel** [comment]
Fixed with v3.2.0
